### PR TITLE
Updated StarDix Data

### PR DESCRIPTION
**Pull request by Peng1104**
## Type of change

- [ ] Add new directory for a server
- [X] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/stardix/background@2x.png
Comment (maaaathis) [inline minecraft_servers/stardix/background@2x.png:1]: Compress this file

File: minecraft_servers/stardix/manifest.json [hunk 1]
```diff
@@ -5,36 +5,70 @@
...
+      "color": "#FF69B4",
+      "command": "/play sv",
+      "url": "https://forum.stardix.com/viewforum.php?f=36",
+      "versions": "1.8<*"
     }
   },
-  "yt_trailer": "UlZpyGrCkTk"
+  "discord": {
+    "server_id": 447833437697474562,
+    "rename_to_minecraft_name": true
```
Comment (maaaathis) [inline minecraft_servers/stardix/manifest.json:60]: `true` is reserved for LabyMod partners only

File: minecraft_servers/stardix/manifest.json [hunk 2]
```diff
@@ -5,36 +5,70 @@
   "server_wildcards": [
     "%.stardix.com"
   ],
+  "supported_languages": [
+    "pt_BR", "pt"
```
Comment (maaaathis) [inline minecraft_servers/stardix/manifest.json:9]: `pt_BR` isn't part of [ISO 639](https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes)

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/stardix/manifest.json (hunk 1 above)
Comment (Peng1104) [reply minecraft_servers/stardix/manifest.json:60]: StarDix is a LabyMod partner server

File: minecraft_servers/stardix/background@2x.png
Comment (Peng1104) [reply minecraft_servers/stardix/background@2x.png:1]: Compressed it

File: minecraft_servers/stardix/manifest.json (hunk 2 above)
Comment (Peng1104) [reply minecraft_servers/stardix/manifest.json]: I have remove it

**Review by DoJapHD — APPROVED** [review]

**Pull request merged**